app/components/incident-summary/incident-summary.tsx

```diff
@@ -121,14 +121,15 @@ export default function IncidentSummary({incident, showForm}: {incident: Inciden
                           <Button data-test="button-edit-summary" type="text" icon={<EditOutlined />} onClick={onSummaryEditClick}>Edit</Button>
                         }
                         >
-                        <strong>Since</strong> <span className="when">{summary.whenUtcString}</span>
+                        <strong className="status">[{summary.status}]</strong>
```

Comment:
I moved it in here because I feel the status is an important part. +my next PR will make it editable 😁